app/assets/javascripts/file_manager.js

```diff
@@ -8,6 +8,21 @@ function injectFileInput() {
 
   new_file_field.change(function() {
     sanitized_filename_check(this);
+    var file_count = 0;
+    var not_terminate = true;
+    while (file_count<this.files.length && not_terminate){
```

Comment:
Identifier 'file_count' is not in camel case.<br>Identifier 'not_terminate' is not in camel case.